jellyfin_kodi/jellyfin/api.py

```diff
@@ -254,37 +254,20 @@ def get_items_theme_song(self, parent_id):
             'ParentId': parent_id
         })
 
-    def get_plugins(self):
-        return self._get("Plugins")
-
-    def check_companion_installed(self):
+    def check_companion_enabled(self):
         """
+        True = Enabled
+        False = Not enabled
         None = Unknown
-        True = Installed, but possibly not loaded right now
-        False = Not installed, scheduled for uninstalling or disabled
         """
         try:
-            kodi_sync_queue = [
-                x
-                for x in self.get_plugins()
-                if x.get("Id") == "771e19d653854cafb35c28a0e865cf63"
-            ]
-
-            LOG.debug("KodiSyncQueue Plugins result: %s", kodi_sync_queue)
-
-            kodi_sync_queue_filtered = [
-                x
-                for x in kodi_sync_queue
-                if x.get("Status")
-                in ["Active", "Restart", "Malfunctioned", "NotSupported"]
-            ]
-
-            if kodi_sync_queue_filtered:
-                return True
-            else:
-                return False
+            plugin_settings = self._get("Jellyfin.Plugin.KodiSyncQueue/GetPluginSettings") or {}
+            return plugin_settings.get('IsEnabled')
+
         except requests.RequestException as e:
```

Comment:
Replacing this with `except HTTPException as e:` should make sure the exception is caught.